src/plugin.ts

```diff
@@ -194,6 +174,15 @@ export function makePlugin<EnvSpecificOptions>(loadOptions: (opts: EnvSpecificOp
           },
         },
 
+        Identifier(path: NodePath<t.Identifier>) {
```

Comment:
this ensures that newly added identifiers do actually reference the import binding. Otherwise ts plugin will remove the import

Reply:
this identifiers are also added by babel import util

Reply:
This is only a problem because we do this in pre

Reply:
This is running in pre as well. So it only affects identifiers added during ast transform

Reply:
i could also have my own custom plugins that do some babel ast changes during template transform